aikau/src/main/resources/alfresco/forms/TabbedControls.js

```diff
@@ -83,9 +83,9 @@ define(["alfresco/layout/AlfTabContainer",
         "dojo/_base/declare",
         "dojo/_base/lang",
         "dojo/_base/array"], 
-        function(HorizontalWidgets, LayoutMixin, declare, lang, array) {
+        function(AlfTabContainer, LayoutMixin, declare, lang, array) {
```

Comment:
Potential breaking change for anyone who has extended TabbedControls, but has no impact on the actual fix, as far as I can tell, so might be worth leaving out of a hotfix?